ewkcoffea/modules/corrections.py

```diff
@@ -586,22 +586,14 @@ def ApplyJetSystematics(year,cleanedJets,syst_var):
         return cleanedJets.JER.up
     elif (syst_var == f'CMS_res_j_{year}Down'):
         return cleanedJets.JER.down
-    elif (syst_var == 'nominal'):
+    elif (syst_var == 'nominal') or (syst_var.startswith(f'CMS_scale_met_unclustered_energy_{year}')):
         return cleanedJets
     elif (syst_var == f'CMS_scale_j_{year}Up'):
         return cleanedJets.JES_Total.up
     elif (syst_var == f'CMS_scale_j_{year}Down'):
         return cleanedJets.JES_Total.down
     else:
-        try:
```

Comment:
Why don't we need this try except anymore?

Reply:
This 'try' method was for the 27 systematics, and I am not sure how it may get handled with new names. Removing it ensures nothing unexpected will happen.